main.md

```diff
@@ -301,6 +301,8 @@ Algorithm" registry [IANA.Hash.Algorithms]. SD-JWTs with hash algorithm
 identifiers not found in this registry are not considered valid and MUST NOT be
 accepted by verifiers.
 
+To promote interoperability among implementations, the SHA-256 hash algorithm is mandatory to implement.
```

Comment:
... the SHA-256 hash algorithm MUST be implemented.

Reply:
Agreed. Perhaps something even shorter like:

Implementations MUST support the SHA-256 hash algorithm as shown in Example 1.

Reply:
Agreed with @peppelinux proposal. It may be sufficient to just say:

Implementations MUST support the SHA-256 hash algorithm ()